### ISSUE TITLE
Doesn't seem to work on anchor

### ISSUE THREAD

**Issue by mellis481** [NicholasBoll/cypress-enter-plugin#2]
I'm working on skip nav link functionality in support of accessibility for my app.  Originally, I was using a `<button data-testid="skip-link">` element and the following Cypress test (using cypress-enter-plugin) worked:

```
cy.get('[data-testid="skip-link"]').should('be.visible').type('{enter}');
```

I recently switched this element to an `<a>` and now the above test does not successfully action the anchor.  Replacing `.type('{enter}')` with `click()` allows the anchor to be actioned, but that obviously doesn't allow me to test accessibility.

Is there any reason this wouldn't work on an anchor?

ps. I've updated the tests to show the failing anchor behavior.  https://github.com/NicholasBoll/cypress-enter-plugin/pull/3

**Comment by mellis481** [comment]
https://github.com/NicholasBoll/cypress-enter-plugin/pull/3

**Comment by NicholasBoll** [comment]
Fixed in `v1.0.1`
